src/pages/books.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+import BookCard from '../components/BookCard';
+import { IBook } from '../interfaces/books';
+
+const Books = () => {
+    const [books, setBooks] = useState<IBook[]>([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const fetchBookData = async () => {
+        try {
+            const response = await fetch('/mockData/books.json');
+            const json = await response.json();
+            setBooks(json);
+        } catch (error) {
+            console.error('Error fetching data:', error);
+        } finally {
+            setIsLoading(false);
+        }
+    };
+
+    useEffect(() => {
+        fetchBookData();
+    }, []);
```

Comment:
Nit - ESList would/should generally be complaining about the lack of `fetchBookData` in the dependency list of `useEffect`. While it is annoying, it should ideally be listened to